rfcs/react-components/convergence/provider-style-overrides.md

```diff
@@ -0,0 +1,132 @@
+# RFC: Provider style overrides
+
+@ling1726
+
+## Summary
+
+Proposes two methods of overriding the default styles on the `FluentProvider`
+
+## Problem statement
+
+The `FluentProvider` ships with [default styles](https://github.com/microsoft/fluentui/blob/656f002f3062c0a7962fb23eb165dec64ecc1509/packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderStyles.ts#L11-L18) which can be too opinionated for some customers, especially those who have
+strict requirements on global styles in their application.
+
+Currently it's not possible to use `makeStyles` to override these default styles. See the example below:
+
+```tsx
+import { makeStyles } from '@fluentui/react-components';
+
+const useProviderOverrides = makeStyles({
+  provider: {
+    // ⚠️ requires marginRight RTL style
+    marginLeft: '1px',
+  },
+});
+
+const App = () => {
+  const styles = useProviderOverrides();
+  return <FluentProvider className={styles.provider}>{children}</FluentProvider>;
+};
+```
+
+The above example is provlematic because the `TextDirectionProvider` is rendered
```

Comment:
we would still need `TextDirectionProvider` because there's no guarantee that folks would use logical properties. If we did decide to enforce this then.... I guess we're going full on v10 :)